src/js/components/Map.js

```diff
@@ -352,6 +352,15 @@ export default class Map extends Component {
         }
       } else if (layer.layerType === 'ArcGISFeatureLayer' && layer.featureCollection && layer.featureCollection.layers) {
         layer.featureCollection.layers.forEach((sublayer) => {
+
+          // We don't want to do this, but we have to...
+          // A user will NEVER know this sublayer.id
+          // The user enters the layer.id in resources.js which DOES NOT
+          // equal the sublayer.id. This messes up the whole app.
+          if (sublayer.id.indexOf(layer.id) > -1) {
+            sublayer.id = layer.id;
+          }
```

Comment:
We should probably remove this since we decided that we won't handle this very unique case